components/gh_events/GitHubEvents.tsx

```diff
@@ -57,3 +51,13 @@ export default function GitHubEvents({ minimal }: { minimal?: boolean }) {
     </div>
   );
 }
+
+async function getEvent() {
+  const response = await octokit.paginate("GET /orgs/{org}/events", {
```

Comment:
We only need the first 5 events. This does not require us to paginate as the first page of 100 results would be sufficient.

```suggestion
  const response = await octokit.request("GET /orgs/{org}/events", {
```